plugins/com.google.cloud.tools.eclipse.ui.util/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,8 @@ Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
 Require-Bundle: org.eclipse.ui.workbench,
- org.eclipse.ui
+ org.eclipse.ui,
+ org.eclipse.search
```

Comment:
My thinking here was that the org.eclipse.search bundle will be always there (or not?) so we could just reuse that, it won't increase the installed size of our plugin.

And probably I should file an issue to add the icon the the Eclipse API, I've seen several plugins containing a copy of the icon.